.github/workflows/test.yml

```diff
@@ -52,13 +52,14 @@ jobs:
       - name: Run Forge build
         run: |
           forge --version
+          forge install OpenZeppelin/openzeppelin-foundry-upgrades
```

Comment:
forge install is supposed to be installed into .gitsubmodule

https://book.getfoundry.sh/reference/forge/forge-install
> Dependencies are installed as git submodules. If you do not want this behavior, pass --no-git.

Is this what you mean? I could not test the workflow since the GHA tests are not running, so not sure if this won't work yet.

Reply:
yes, I meant we might checkin the `.gitsubmodule` which contains `openzeppelin-foundry-upgrades`, so that we can just run "forge install" to install all libraries.

Reply:
I'm realizing my install didn't generate .gitsubmodule, just added this to .gitmodules

```
[submodule "lib/openzeppelin-foundry-upgrades"]
path = lib/openzeppelin-foundry-upgrades
url = https://github.com/OpenZeppelin/openzeppelin-foundry-upgrades
```

Let's see if it works

Reply:
Ah, yes, it should be `.gitmodules` but `.gitsubmodule`, it was my typo.
btw, with the `.gitmodules`, we might remove this line below as it will automatically installed when run `forge build`.
`forge install OpenZeppelin/openzeppelin-foundry-upgrades`

Reply:
I had to keep the install line, since it won't install with only `.gitmodules`
I did add the option `--no-git` because we .gitignored `lib` and the install failed.

Reply:
Looks like our GHA has some issue with `.gitmodules` to install dependencies. but it is not a blocking issue, we can address this later.